### PR TITLE
Updating Gazelle version in example

### PR DESCRIPTION
**Pull request by chrislovecnm**
Bumping the version of Gazelle in the WORKSPACE file.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

Both of the above are not applicable.

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [X] Other... Please describe:

Updating the version of Gazelle that is used in the example.

## What is the current behavior?

We are using an old version
Issue Number: N/A

## What is the new behavior?
We are using the current version.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by chrislovecnm** [comment]
So buildifier is failing its test, but I have run buildifier locally, and it is not making any changes.

File: examples/build_file_generation/WORKSPACE [hunk 1]
```diff
@@ -12,31 +12,28 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
...
-        # No release since March, and we need subsequent fixes
-        "https://github.com/bazelbuild/bazel-gazelle/archive/41b542f9b0fefe916a95ca5460458abf916f5fe5.zip",
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-gazelle/releases/download/v0.27.0/bazel-gazelle-v0.27.0.tar.gz",
+        "https://github.com/bazelbuild/bazel-gazelle/releases/download/v0.27.0/bazel-gazelle-v0.27.0.tar.gz",
     ],
 )
 
-load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
 load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
```
Comment (fmeum) [inline examples/build_file_generation/WORKSPACE:32]:
```suggestion
load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
```

should fix the buildifier finding

Comment (chrislovecnm) [reply examples/build_file_generation/WORKSPACE]: ` buildifier --lint=fix --warnings=native-py -warnings=all` fixed it :)

**Review by groodt — APPROVED** [review]
LGTM

**Pull request merged**